package.json

```diff
@@ -22,8 +22,7 @@
   },
   "repository": "https://github.com/emberjs/ember-inspector",
   "engines": {
-    "node": ">= 0.10.0",
-    "npm": ">= 2.0.0"
+    "node": ">= 0.10.0"
   },
   "emberVersionsSupported": [
```

Comment:
`emberVersionsSupported`: The range of Ember versions supported by current master. It's an array, first item is "from", second item is "to" which is an Empty string because it's 2.7.0+.
`previousEmberVersionsSupported`: all the previous builds and what Ember versions they support. These versions are only included in the final build before deployment.